pkg/common/scheduler/statefulset/snapshot.go

```diff
@@ -22,6 +22,10 @@ import "go.uber.org/zap"
 type Snapshot struct {
 	// free tracks the free capacity of each pod.
 	free map[string]int32
+
+	// lastOrdinal is the ordinal index corresponding to the last statefulset replica
+	// with placed vpods.
+	lastOrdinal int32
```

Comment:
Is the adding of lastOrdinal only for snapshot test?

Reply:
It's is used below, to make sure no pods with placed vpods are killed.